test/unit/actions-test.js

```diff
@@ -1,4 +1,6 @@
+import 'babel-polyfill';
```

Comment:
Yeah probably. I just continued what we currently had to not make this PR any bigger.